vermouth/tests/test_ff_files.py

```diff
@@ -513,6 +513,68 @@ def test_param_effector_fail():
         with pytest.raises(IOError):
             vermouth.ffinput.read_ff(lines, ff)
 
+    @staticmethod
+    @pytest.mark.parametrize('interaction_lines, expected', (
+        [
+            """
+            [ dihedrals ]
+            BB SC1 SC2 SC3 2 1 2 3 4
+            BB SC1 SC2 SC3 1
+            """,
+            {'impropers': [vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC3'.split(),
+                                                         parameters='2 1 2 3 4'.split(),
+                                                         meta={})],
+             'dihedrals': [vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC3'.split(),
+                                                         parameters=['1'],
+                                                         meta={})]}
+        ],
+        [
+            """
+            [ dihedrals ]
+            BB SC1 SC2 SC3 1 1 2 3 4
+            BB SC1 SC2 SC3 1
+            """,
+            {'dihedrals': [vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC3'.split(),
+                                                         parameters='1 1 2 3 4'.split(),
+                                                         meta={}), 
+                           vermouth.molecule.Interaction(atoms='BB SC1 SC2 SC3'.split(),
+                                                         parameters=['1'],
+                                                         meta={})],
+             'impropers': []}
```

Comment:
Jon is right, at least this test should fail. The default behavior is interactions overwrite each other if they have the same atom names involved.